www/src/scss/_overlay.scss

```diff
@@ -157,6 +157,52 @@ input[type='color'] {
   width: 60px;
 }
 
+.overlay-off {
+  .cp-icon,
+  .setting-label {
+    color: #ddd;
+    cursor: not-allowed;
+    pointer-events: none;
```

Comment:
minor thing: larivaar/padched button and copy url button still seem to work when disabled.